### DISCUSSION TITLE
Currency: How do I add South African currency instead of USD, I want ZAR

### DISCUSSION THREAD

**Discussion by valduze** [strapi/strapi#7068] [Legacy Topics]
Hi Guys,
I need to know if Strapi support my currency which is ZAR, if it does, can you please point me on documentation on how to implement that.
Thanks,
Jongi

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.
